pkg/defaulting/cluster.go

```diff
@@ -94,6 +94,11 @@ func DefaultClusterSpec(ctx context.Context, spec *kubermaticv1.ClusterSpec, tem
 		return fieldErr
 	}
 
+	// Set the audit logging settings
+	if seed.Spec.AuditLogging != nil {
+		spec.AuditLogging = seed.Spec.AuditLogging
```

Comment:
let's make a real copy 

```suggestion
		spec.AuditLogging = new(AuditLoggingSettings)
		(*seed.Spec.AuditLogging).DeepCopyInto(*spec.AuditLogging)
```